src/main/antlr/OpenDistroSqlParser.g4

```diff
@@ -324,20 +336,30 @@ aggregateWindowedFunction
     | COUNT '(' aggregator=DISTINCT functionArgs ')'
     ;
 
+functionAsAggregatorFunction
+    : (AVG | MAX | MIN | SUM)
+      '(' aggregator=(ALL | DISTINCT)? functionCall ')'
+    | COUNT '(' aggregator=(ALL | DISTINCT)? functionCall ')'
+    ;
+
 scalarFunctionName
     : functionNameBase
     ;
 
 functionArgs
-    : (constant | fullColumnName | functionCall | expression)
```

Comment:
As my understanding, the changes are temporary and may be reverted later. Could you comment it out rather than remove it and put new rules you added together (probably better with comments explaining why)? Just in case someone else would make change on top of yours and hard to track only by commit history.

Reply:
Ok that makes sense, will do. Thanks!